### PR TITLE
fix: add missing zone_sync_ssl_conf_command directive

### PR DESCRIPTION
**Pull request by dylan-way**
### Proposed changes

Add missing `zone_sync_ssl_conf_command` directive

https://nginx.org/en/docs/stream/ngx_stream_zone_sync_module.html

### Checklist

Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [`CONTRIBUTING`](https://github.com/nginxinc/nginx-go-crossplane/blob/main/CONTRIBUTING.md) document
- [x] If applicable, I have added tests that prove my fix is effective or that my feature works
- [x] If applicable, I have checked that any relevant tests pass after adding my changes
- [x] I have updated any relevant documentation ([`README.md`](https://github.com/nginxinc/nginx-go-crossplane/blob/main/README.md))

### CODE REVIEW

**Review by valyria257 — APPROVED** [review]

File: analyze_test.go [hunk 1]
```diff
@@ -377,6 +377,131 @@ func TestAnalyzeFlagArgs(t *testing.T) {
...
+		"zone_sync_ssl_conf_command invalid context": {
+			&Directive{
+				Directive: "zone_sync_ssl_conf_command",
+				Args:      []string{"somename", "somevalue"},
+				Line:      5,
+			},
+			blockCtx{"http", "server"},
+			true,
+		},
+		"zone_sync_ssl_conf_command invalid args": {
```
Comment (russokj) [inline analyze_test.go:478]: Any need to test 0 args?

File: analyze_test.go [hunk 2]
```diff
@@ -377,6 +377,131 @@ func TestAnalyzeFlagArgs(t *testing.T) {
...
+func TestAnalyze_zone_sync(t *testing.T) {
+	t.Parallel()
+	testcases := map[string]struct {
+		stmt    *Directive
+		ctx     blockCtx
+		wantErr bool
+	}{
+		"zone_sync in stream server context ok": {
+			&Directive{
+				Directive: "zone_sync",
```
Comment (russokj) [inline analyze_test.go:390]: I assume these are just missing unit tests for other directives unrelated to the `zone_sync_ssl_conf_command`.

File: analyze_test.go (hunk 1 above)
Comment (dylan-way) [reply analyze_test.go]: Sure, I just added this case.

File: analyze_test.go (hunk 2 above)
Comment (dylan-way) [reply analyze_test.go:390]: Yes, I didn't add tests to exhaustively check all of the `zone_sync...` directives, but I spot checked a few alongside the one I'm adding here.

**Review by ornj — APPROVED** [review]

**Review by dakshinai — APPROVED** [review]

**Pull request merged**